versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/UseLatestVersionsMojo.java

```diff
@@ -62,7 +45,7 @@ public class UseLatestVersionsMojo extends UseLatestVersionsMojoBase {
      * @since 1.2
      */
     @Parameter(property = "allowMajorUpdates", defaultValue = "true")
-    private boolean allowMajorUpdates = true;
+    protected boolean allowMajorUpdates = true;
```

Comment:
if we introduce also geters for it I would like to left it private

Reply:
I actually exposed it for the tests, so that we wouldn't need to use reflection to manipulate those.